src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -143,17 +145,19 @@ public synchronized void handleMessage(CorfuMsg msg, ChannelHandlerContext ctx,
                 if (req.getNumTokens() == 0) {
                     long max = 0L;
                     boolean hit = false;
-                    ImmutableMap.Builder<UUID, Long> streamsLastIssued = ImmutableMap.builder();
+                    ImmutableMap.Builder<UUID, Long> latestStreamOffsets = ImmutableMap.builder();
```

Comment:
Looks like the purpose of latestStreamOffset is to captures from lastLocalOffsetMap all the lastOffset values for streams in this request, correct? please comment

and .. can you choose a  name to better reflect that difference?